prime-router/src/main/kotlin/cli/SettingCommands.kt

```diff
@@ -106,20 +101,7 @@ abstract class SettingCommand(
     enum class SettingType { ORGANIZATION, SENDER, RECEIVER }
 
     val jsonMapper = JacksonMapperUtilities.allowUnknownsMapper
-    val yamlMapper: ObjectMapper = ObjectMapper(YAMLFactory()).registerModule(
-        KotlinModule.Builder()
-            .withReflectionCacheSize(512)
-            .configure(KotlinFeature.NullToEmptyCollection, false)
-            .configure(KotlinFeature.NullToEmptyMap, false)
```

Comment:
It doesn't look like `JacksonMapperUtilities.yamlMapper` has these configured... is that ok?

Reply:
those are defaults, so no need to say it again